src/Sylius/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -59,35 +52,29 @@ public function showAction(Request $request)
 
     /**
      * @param Request $request
-     * @param string  $namespace
      *
      * @return Response
      */
-    public function updateAction(Request $request, $namespace)
+    public function updateAction(Request $request)
     {
-        $this->isGrantedOr403($namespace);
+        $schemaAlias = $request->get('schema');
```

Comment:
That should come from request attributes I think, not from query/post. ``$request->attributes->...``?